app/scripts/controllers/overview.js

```diff
@@ -692,10 +697,82 @@ function OverviewController($scope,
     return true;
   };
 
+  // part of lodash since 4.0.0
+  function byKey(selector) {
+    return function (accum, item) {
+      var key = selector(item)
+      accum[key] = item;
+      return accum;
+    }
+  }
+
+  function getOvmId(vm) {
+    var ownerReferences = vm.metadata.ownerReferences;
+    if (!ownerReferences) {
+      return null;
+    }
+    return _(ownerReferences)
+      .filter(function (reference) {
+        return reference.kind === 'OfflineVirtualMachine';
+      })
+      .map(function (reference) {
+        return reference.uid;
+      })
```

Comment:
You should be able to use the shorthand here:

```js
.filter({ kind: 'OfflineVirtualMachine' })
.map('uid')
```

Reply:
done